### PR TITLE
readme review updates

### LINKED ISSUE #25: Readme review
- DijitWrapper example
   - `_onCalendarChange` does not need to be an arrow function
- ReduxInjector example
   - Perhaps change the `imports` to use the default exports

### PR DESCRIPTION
**Pull request by tomdye**
The following has been addressed in the PR:

* [ ] There is a related issue
* [ ] All code has been formatted with [`prettier`](https://prettier.io/) as per the [readme code style guidelines](./../#code-style)
* [ ] Unit or Functional tests are included in the PR

**Description:**

Readme review updates

Resolves #25

### CODE REVIEW

**Comment by tomdye** [comment]
@agubler forgot to skip ci, sorry

File: README.md [hunk 1]
```diff
@@ -64,11 +83,11 @@ An injector can be defined in the registry, which is then provided to the `Proje
 
 ```typescript
 import global from '@dojo/shim/global';
-import { ProjectorMixin } from '@dojo/widget-core/mixins/Projector';
-import { ReduxInjector } from '@dojo/interop/redux/ReduxInjector';
-import { Registry } from '@dojo/widget-core/Registry';
+import ProjectorMixin from '@dojo/widget-core/mixins/Projector';
```
Comment (agubler) [inline README.md:86]: lol 😉

Comment (tomdye) [reply README.md:86]: ?

Comment (dylans) [reply README.md:86]: Could we resolve this?

**Review by dylans — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (agubler) [reply README.md:86]: it's resolved, don't worry

**Review by agubler — APPROVED** [review]

**Review by dylans — APPROVED** [review]

**Pull request merged**